diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index c8a134e..65d4558 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1115,13 +1115,13 @@ if( CMAKE_SYSTEM_NAME MATCHES "Windows" )
    )
 
    # Copy the VC runtime libraries as well
-   add_custom_command(
-      TARGET
-         ${TARGET}
-      COMMAND
-         ${CMAKE_COMMAND} -E copy ${CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS} ${_DEST}
-      POST_BUILD
-   )
+   #add_custom_command(
+   #   TARGET
+   #      ${TARGET}
+   #   COMMAND
+   #      ${CMAKE_COMMAND} -E copy ${CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS} ${_DEST}
+   #   POST_BUILD
+   #)
 
    # Copy resource files into build folder
    add_custom_command(
